### ISSUE TITLE
NEOGEO X Arcade Stick [USB]

### ISSUE THREAD

**Issue by phantasum** [Maschell/hid_to_vpad#35]
Hello Group,

I've attempted at creating a .ini file using a text editor (on mac). Unfortunately, I get no response and the application does not even boot up when the file is the ://wiiu/controller directory. I get a black screen and it appears to crash the Wii U. I would appreciate any assistance. 

At the sametime I know this type of pad is going be quite common/popular in the future considering you can find them quite cheaply and they as remarkably similar to the original SNK sticks.

Thanks!

Below is a copy of the .ini file 
(file name is NEOGEO_Arcade_Stick.ini):
----------------------------------------------------------------------------------------------------------------
[vid=0x1292,pid=0x4e47]

//DPad
DPAD_MODE=DPAD_Normal
VPAD_BUTTON_DPAD_ABS_UP=0x02,0x00
VPAD_BUTTON_DPAD_ABS_DOWN=0x02,0x04
VPAD_BUTTON_DPAD_ABS_LEFT=0x02,0x06
VPAD_BUTTON_DPAD_ABS_RIGHT=0x02,0x02

//Buttons
VPAD_BUTTON_A=0x00,0x01
VPAD_BUTTON_B=0x00,0x02
VPAD_BUTTON_X=0x00,0x08
VPAD_BUTTON_Y=0x00,0x04
VPAD_BUTTON_PLUS=0x01,0x02
VPAD_BUTTON_MINUS=0x01,0x01
PAD_COUNT=1

**Comment by Maschell** [comment]
Someone else had a similar problem with his .ini file

I tried both, and had no issues (at loading).
Did you try a fresh SD card?
Here is the ini I tested

[neo_geo.zip](https://github.com/Maschell/hid_to_vpad/files/1028494/neo_geo.zip)

**Comment by phantasum** [comment]
For reference. I'm using a SanDisk Ultra 64GB Class 10.
HID to VPAD - nightly (v0.9j-nightly-cf54450

Used your file on the original SD card, had a black screen(freeze/crash).
Used your file on different SD card (Sandisk 16GB), still crashed/froze.

Might be something in the way Mac OS does file storage and the software interprets the .ini file type.
Could I suggest an auto HID set up for HIDtoVPAD so a person with a USB HID device could just associate control prompts to the button presses. That or adding a feature within HID Test that allows for creation of ini files.

Thanks for all your help.

**Comment by CreeperMario** [comment]
> I've attempted at creating a .ini file using a text editor (on mac)

Which text editor did you use? TextEdit, even if it is in plain text mode, replaces punctuation marks like apostrophes and quotation marks (`'` and `"`) with the appropriate Unicode characters, rather than ASCII. This causes issues in software that expect ASCII characters. (Even GCC yells at me if I use TextEdit.) I would recommend using another plain text editor.

**Comment by phantasum** [comment]
I don't believe its is an issue at this point. Since I used the file Maschell created, the he tested and claimed worked(see second post).

**Comment by Maschell** [comment]
The newest version has more checks while parsing config files.
At this shouldn't crash anymore with "faulty" files. Can you confirm it?
